src/TParsec/Running.idr

```diff
@@ -37,6 +37,9 @@ MonadRun Maybe where
 MonadRun Identity where
   runMonad (Id a) = pure a
 
+Monoid s => MonadRun (StateT s Identity) where
+  runMonad st = pure $ evalState st neutral
+
```

Comment:
Feels wrong to require an entire `Monoid` when we only care about the fact the set is
`Pointed` (I guess the advantage is that contribs already mentions `Monoid` but not `Pointed`).

Reply:
Yeah I don't think there's a pre-existing `Pointed`.